content/writing/digital-approaches-for-social-change-organisations.mdx

```diff
@@ -0,0 +1,112 @@
+
+Our work was intended to be speculative: broaching what could be just over the horizon beyond the obvious next steps. However, it also needed to be concrete enough for JRF staff to be able to gather real feedback from service users, understand how to get there and to advocate internally for our proposed approach.
+
+### What we did
+
+- Conducted some initial discovery work to familiarise ourselves with the project.
+- Ran workshops with their team to reflect upon the work they'd already done and identify challenges, needs and opportunities.
+- Conducted research interviews with JRF staff and external stakeholders like organisers and researchers.
+- Created a series of prototypes and tested them with members of the general public.
+- Ran show and tell sessions with the wider organisation to share what we were learning and gather feedback.
+
+![Screenshots of the different levels of fidelity that we prototype in: from lightweight wireframes to Figma designs to a high fidelity prototype in code](../../assets/jrf-prototypes-fidelity.png)
+
+We produced a range of provocations at varying levels of fidelity: rough sketches and wireframes, flat Figma prototypes to suggest look and feel, functional prototypes that explored affordances of code.
+
+Many of the ideas sparked through our research were more deeply organisational than superficially digital. We strongly believe that digital artefacts express the organisational culture, values and methodology that produces them. Our challenge was to keep trying to hone in on things that we could tangibly demonstrate through digital prototypes that, like the tip of the iceberg, would coincide with deeper changes within the organisation.
+
+## Core themes
+
+We begun our design process by asking ourselves: how might we make JRF's extensive research into poverty more accessible to a broad audience? We looked at ways to enable anyone – whether that's campaigners, people with lived experience of poverty, or individuals wanting to take action – to find the most relevant information to them and (crucially) to use it in their organising efforts.
+
+Our initial wireframes explored a broad range of ideas, loosely grouped under the theme "data for action". We explored how to weave other voices – either grassroots campaigners or people with lived experience – into JRF's reports, how to contextualise datasets through postcode search, and how to slice up their research into easily shareable and exportable chunks.
+
+At the same time, we were conducting research interviews with organisers. From this, and the feedback we gathered in our show-and-tell sessions, the idea of relationship-building emerged as the most crucial question. JRF have already built relationships with grassroots organisations and individuals, but there's so much potential to deepen these relationships and form new ones.
```

Comment:
Maybe throw in some examples of who we talked to here. Especially the ones that JRF are interestingly already working with to produce their reports.